### ISSUE TITLE
[Egg Request]: L4D2

### ISSUE THREAD

**Issue by Red-Banana-Official** [QuintenQVD0/Q_eggs#65]
### Does this expand an already existing service

No

### Link to a game or software

https://steamdb.info/app/550/

### Links for the download

https://store.steampowered.com/app/550/Left_4_Dead_2/

### Links for the install docs

Umm.. not sure, couldn't find any.. i do think it will work on ARM with adjustments

### Does this app work on ARM?

I don't know

**Comment by Truepanelhosting** [comment]
WARNING: very new to this, i am just trying to help.

Got the server to run on ARM, using ur docker image.

**Some issues:**
dlmopen steamservice.so failed: Cannot dlopen("steamservice.so"/0xddf80028, 2)

**Multiple failed lines like these**
failed to dlopen /home/container/left4dead2_dlc3/bin/matchmaking_ds_srv.so error=Cannot dlopen("/home/container/left4dead2_dlc3/bin/matchmaking_ds_srv.so"/0xf70d867c, 2)

Steamservice.so seems fine at first:
[S_API] SteamAPI_Init(): Loaded '/home/container/.steam/sdk32/steamclient.so' OK.

But fails later:
dlmopen steamservice.so failed: Cannot dlopen("steamservice.so"/0xddf80028, 2)

**Comment by QuintenQVD0** [comment]
> WARNING: very new to this, i am just trying to help.
> 
> Got the server to run on ARM, using ur docker image.
> 
> **Some issues:** dlmopen steamservice.so failed: Cannot dlopen("steamservice.so"/0xddf80028, 2)
> 
> **Multiple failed lines like these** failed to dlopen /home/container/left4dead2_dlc3/bin/matchmaking_ds_srv.so error=Cannot dlopen("/home/container/left4dead2_dlc3/bin/matchmaking_ds_srv.so"/0xf70d867c, 2)
> 
> Steamservice.so seems fine at first: [S_API] SteamAPI_Init(): Loaded '/home/container/.steam/sdk32/steamclient.so' OK.
> 
> But fails later: dlmopen steamservice.so failed: Cannot dlopen("steamservice.so"/0xddf80028, 2) @QuintenQVD0

what image did you set a server image?

if you use the emulation one then this game my guess to complex, I will have to use the FEX dev image

**Comment by Truepanelhosting** [comment]
quintenqvd\/pterodactyl_images:emulation is the image i used, i can try it with the FEX dev image too.

**Comment by Truepanelhosting** [comment]
I tried making some adjustments, but it didn't work.

**Comment by QuintenQVD0** [comment]
> I tried making some adjustments, but it didn't work. Would you be able to provide me with the image for fex?

Its out there. Look for example in the fivem egg.

**Comment by Truepanelhosting** [comment]
> > I tried making some adjustments, but it didn't work. Would you be able to provide me with the image for fex?
> 
> Its out there. Look for example in the fivem egg.

Yeah i got the image, but i guess it will need a custom install script?

**Comment by QuintenQVD0** [comment]
> > > I tried making some adjustments, but it didn't work. Would you be able to provide me with the image for fex?
> > 
> > 
> > Its out there. Look for example in the fivem egg.
> 
> Yeah i got the image, but i guess it will need a custom install script?

the install script will need to be modified indeed

**Comment by QuintenQVD0** [comment]
> > > > > I tried making some adjustments, but it didn't work. Would you be able to provide me with the image for fex?
> > > > 
> > > > 
> > > > Its out there. Look for example in the fivem egg.
> > > 
> > > 
> > > Yeah i got the image, but i guess it will need a custom install script?
> > 
> > 
> > the install script will need to be modified indeed
> 
> That will be beyond what im experienced in.. ive checked for simulator scripts, but to no avail. Would you be able to provide me with a install script? Or direct me to a place where i can find the information needed to make it myself?
> 
> Thanks! :)

I normaly have time this evening to test and make my own egg

**Comment by Truepanelhosting** [comment]
That is fine.. 
I mean, i think i can get it to work with the right install script.. ill try and see what i can find for that. Its a fun thing to work on
